x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -212,7 +207,7 @@ const UnstyledProcessEventDot = React.memo(
       isLabelFilled,
       labelButtonFill,
       strokeColor,
-    } = cubeAssetsForNode(isProcessTerminated, isProcessOrigin);
+    } = cubeAssetsForNode(isProcessTerminated, false);
```

Comment:
this isn't defined yet. we dont have a way to know if its a trigger? TODO add comment